backend/lp_generator.py

```
@@ -144,60 +142,44 @@ def generate_image_by_imagen3(prompt, file_name, aspect_ratio=None):
-        except Exception as e:
-            print(f"Base64デコードエラー: {e}")
-            # Base64デコードに失敗した場合、直接試す
-            image = Image.open(BytesIO(image_bytes))
-    else:
-        # バイナリデータとして直接処理
-        image = Image.open(BytesIO(image_bytes))
-    image.save(file_name)
-    print(f"画像を保存しました: {file_name}")
-    return file_name
+            font = ImageFont.load_default()
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)